test/libyul/yulSyntaxTests/tstore_warning_only_once.yul

```diff
@@ -0,0 +1,12 @@
+{
+    function f() {
+        tstore(0, 0)
+        tstore(0, 1)
+        let x := tload(0)
+        tstore(1, 0)
+    }
+}
+// ====
+// EVMVersion: >=cancun
+// ----
+// Warning 2394: (29-35): Transient storage as defined by EIP-1153 can break the composability of smart contracts: Since transient storage is cleared only at the end of the transaction and not at the end of the outermost call frame to the contract within a transaction, your contract may unintentionally misbehave when invoked multiple times in a complex transaction. To avoid this, be sure to clear all transient storage at the end of any call to your contract. The use of transient storage for reentrancy guards that are cleared at the end of the call is safe.
```

Comment:
We can also remove the source location. At the moment, it is showing the location of the first occurrence of `tstore`.

Reply:
I guess it is fine to leave the first occurrence, but I'm not against removing it either.

Reply:
Users really don't like mystery errors/warnings without a location. I would not invent one artificially if we don't have one, but here pointing at the location of one of the `tstore`s makes perfect sense, even if it's not the only one. We should use it. This way e.g. an IDE will be able to take you to one of the relevant locations and highlight the line as causing the problem.